internal/coreinternal/attraction/attraction.go

```diff
@@ -231,6 +248,20 @@ func NewAttrProc(settings *Settings) (*AttrProc, error) {
 			}
 			action.Regex = re
 			action.AttrNames = attrNames
+		case CONVERT:
+			if valueSourceCount > 0 || a.RegexPattern != "" {
+				return nil, fmt.Errorf("error creating AttrProc. Action \"%s\" does not use value sources or \"pattern\" field. These must not be specified for %d-th action", a.Action, i)
+			}
+			switch a.ConvertedType {
+			case "string":
```

Comment:
I think as a general good practice the literals for the supported converted types could be made constants